test/File.js

```diff
@@ -236,13 +236,13 @@ describe('File', function() {
 
       var file = new File(options);
 
-      var copy1 = file.clone({ contents: false });
+      var copy1 = file.clone({contents: false});
```

Comment:
The rule in question is [`object-curly-spacing`](http://eslint.org/docs/rules/object-curly-spacing), found in [**gulpjs/eslint-config-gulp** /eslint.json:23](https://github.com/gulpjs/eslint-config-gulp/blob/38080cc98d9d9129f122525872774043ec688444/eslint.json#L23)